homeassistant/components/vallox/sensor.py

```diff
@@ -127,6 +130,27 @@ def native_value(self) -> StateType:
         return VALLOX_CELL_STATE_TO_STR.get(super_native_value)
 
 
+class ValloxProfileDurationSensor(ValloxSensorEntity):
+    """Child class for profile duration reporting."""
+
+    @property
+    def native_value(self) -> StateType:
+        """Return the value reported by the sensor."""
+        minutes_left: int | float | None = 0
+
+        if Profile.EXTRA is self.coordinator.data.profile:
```

Comment:
Hold on. I am moving this block to the lib.

Reply:
Use new method from https://github.com/yozik04/vallox_websocket_api/releases/tag/5.3.0

Reply:
Could be an idea to bump the library version in a separate PR I guess.

Reply:
yes, lib version update needs to go in a separate PR.

Reply:
created #120395 for it